.github/workflows/yaks-tests.yaml

```diff
@@ -42,7 +42,7 @@ concurrency:
 
 env:
   YAKS_VERSION: 0.16.0
-  YAKS_RUN_OPTIONS: "--timeout=15m --local -e YAKS_CAMELK_MAX_ATTEMPTS=10 -e YAKS_JBANG_CAMEL_VERSION=4.0.0 -e YAKS_JBANG_KAMELETS_VERSION=4.0.0-SNAPSHOT -e YAKS_JBANG_KAMELETS_LOCAL_DIR=../../../kamelets -e YAKS_KAMELET_API_VERSION=v1alpha1"
```

Comment:
Question regarding `-e YAKS_KAMELET_API_VERSION=v1alpha1` 

Is Camel 4.0.0 able to handle Pipe resources? Last time I checked this raised me some errors because Pipe has not been recognized by Camel JBang

Reply:
I think it is supported completely from 4.0.1

Reply:
https://github.com/apache/camel-kamelets/pull/1566#issuecomment-1689855294

Reply:
yes, 4.0.1 should handle it

I updated `YAKS_JBANG_CAMEL_VERSION=4.0.0` to `YAKS_JBANG_CAMEL_VERSION=4.0.1`